oven

```diff
@@ -264,7 +271,7 @@ function prepare_chroot() {
     arm_binfmt+='/usr/bin/qemu-arm-static:'
 
     cp "$opt_qemu_src" "${opt_mount_point}/${opt_qemu_dst}"
-    if ! ls /proc/sys/fs/binfmt_misc/qemu-arm &> /dev/null ; then
+    if ! ls /proc/sys/fs/binfmt_misc/qemu-arm &> /dev/null ; then       
```

Comment:
Please remove the trailing spaces.

Reply:
Ok, sounds good. I'll revise this shortly.